README.md

```diff
@@ -38,6 +38,19 @@ Status of the toolbox should be updated to wiki https://github.com/gtkfi/ArcSDM/
 
 Development has started by GTK to migrate this tool for ArcGIS Pro 3.0+ (ArcSDM 6.0)
 
+## Usage of the ArcSDM Python env
+
+At the moment is this advised to clone the ArcGIS Pro Python env and downloading the necessary packages to the cloned Python env.
```

Comment:
Great to have this in the readme 👍 

As already discussed on Teams, please include the specific packages that need to be installed as well.

For scikit-learn, the newest version wasn't compatible, so I used v.1.4.2. For tensorflow, the latest (v.2.13) was fine. The package imbalanced-learn didn't seem to be necessary - I didn't install it, and everything worked fine. (Not sure if tensorflow was really needed either 😄)

Reply:
Scikit-learn is required for PCA. Other tools are required for future development. Hence, Tensorflow and Imbalanced-learn are needed.